Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -250,7 +250,7 @@ - (CVMetalTextureRef)getCameraTexture:(CVPixelBufferRef)pixelBuffer plane:(int)p
  Checks whether the camera UVs need to be updated based on the orientation and size of the view port, and updates them if necessary.
  @return True if the camera UVs were updated, false otherwise.
 */
-- (Boolean)checkAndUpdateCameraUVs:(ARFrame *)frame {
```

Comment:
I'm not sure you should change this convention in ObjC files. It seems to be the default for Xcode. That said, I don't have much experience here.

Reply:
Yeah I don't mind either way, but the standard convention I've seen is to have the space.